docs/configuration/environment.md

```diff
@@ -2,10 +2,15 @@
 
 uv accepts the following command-line arguments as environment variables:
 
+- `UV_INDEX`: Equivalent to the `--index` command-line argument. If set, uv will use this URL as the
+  base index for searching for packages.
+- `UV_DEFAULT_INDEX`: Equivalent to the `--default-index` command-line argument. If set, uv will use
+  this space-separated list of URLs as additional indexes when searching for packages.
```

Comment:
Can't be multiple values right? I think these might be roughly swapped but both need some work.